pkgs/applications/audio/openutau/default.nix

```diff
@@ -47,8 +61,8 @@ buildDotnetModule rec {
   # needed until upstream bumps to dotnet 7
   postPatch = ''
     substituteInPlace OpenUtau/OpenUtau.csproj OpenUtau.Test/OpenUtau.Test.csproj --replace \
-      "<TargetFramework>net6.0</TargetFramework>" \
-      "<TargetFramework>net7.0</TargetFramework>"
+      '<TargetFramework>net6.0</TargetFramework>' \
+      '<TargetFramework>net7.0</TargetFramework>'
```

Comment:
why did you change this?

Reply:
That's a good question. I don't remember changing that...

But I'm surprised I used double quotes initially anyway, since I prefer single quotes in bash for any strings that don't need variable or command substitution, so I assume I probably instinctively made it as a drive-by change while looking at the file?

I can revert that change if preferred